cmd/svcat/command/command.go

```diff
@@ -93,3 +109,29 @@ func DetermineNamespace(flags *pflag.FlagSet, currentNamespace string) string {
 
 	return currentNamespace
 }
+
+// AddOutputFlags adds common output flags to a command that can have variable output formats.
+func AddOutputFlags(flags *pflag.FlagSet) {
+	flags.StringP(
+		"output",
+		"o",
+		"",
+		"The output format to use. Valid options are table, json or yaml. If not present, defaults to table",
+	)
+}
+
+func determineOutputFormat(flags *pflag.FlagSet) (string, error) {
+	format, _ := flags.GetString("output")
+	format = strings.ToLower(format)
+
+	switch format {
+	case "", "table":
+		return "table", nil
+	case "json":
+		return "json", nil
+	case "yaml":
+		return "yaml", nil
+	default:
+		return "", fmt.Errorf("unknown output format: %s", format)
```

Comment:
I think it would help to mention the flag and acceptable formats, maybe something like this?

> invalid --output format %q, allowed values are table, json and yaml